pkg/porter/explain.go

```diff
@@ -187,11 +189,17 @@ func (p *Porter) printBundleExplain(o ExplainOpts, pb *PrintableBundle) error {
 	}
 }
 
-func generatePrintable(bun bundle.Bundle, action string) (*PrintableBundle, error) {
+func generatePrintable(bun bundle.Bundle) (*PrintableBundle, error) {
+	stamp, err := configadapter.LoadStamp(bun)
```

Comment:
Instead of returning an error, one way to deal with non-porter created bundles is to set Stamp to an empty struct when it can't be loaded, e.g. `Stamp{}`. Then only print "Porter Version" in the output if it's not empty.

You can make a duplicate of your new test where the stamp isn't set to validate that it's not included in the output.

Reply:
This is for handling the non-Porter bundle by assigning Stamp's empty struct when it returns error.